### DISCUSSION TITLE
Using System.IO and Networking APIs

### DISCUSSION THREAD

**Discussion by spb229** [spatialsys/spatial-unity-sdk#11] [Help] [answered]
Hi is it possible to turn on access to Any System.IO APIs and Networking APIs (UnityWebRequest www) in a custom built Spatial environment? Thanks.

**Comment by AlanSpatial** [comment, marked as answer]
# UnityWebRequest
Currently we don't support any form of web request from inside unity. We plan to add support eventually, but unsure if that will be through enabling the UnityWebRequest or if we will supply a custom solution.

# System.io
Can you explain exactly what API's you want? We are looking at supporting things like `streamReader` _right now_. But any api that can save to, create, or open arbitrary files will not be supported.
